pyproject.toml

```diff
@@ -17,14 +17,15 @@ dependencies = [
     "seaborn>=0.13.2",
```

Comment:
@mfs4rd Whenever we change `pyproject.toml` we should update the lockfile `pdm.lock` with the exact dependencies used.